app/assets/javascripts/app/views/help_view.js

```diff
@@ -133,6 +136,32 @@ app.views.Help = app.views.StaticContentView.extend({
     this.$('#faq').append(help_section.render().el);
   },
 
+  /**
+   * Returns the jQuery <a> element whose data-section property equals the given query
```

Comment:
Finding good name has nothing to do with the language you're writing in.

> `findElByDataSection`

Please reread my comment, this does still not describe what it does and has the second issue still.